Source/Scene/Model.js

```diff
@@ -119,7 +119,7 @@ define([
 
         this.decompressedViewsToCreate = new Queue();
         this.decompressedViews = {};
-        this.decompressionInFlight = 0;
+        this.decompressionInFlight = false;
```

Comment:
Okay, it now uses all available CPUS in a round-robin fashoin.  We could further improve this by checking the number of active tasks and scheduling the task on the "least active" scheduler, but I need to make unrelated changes to Cesium for that to happen (which I'll do in a future PR into master).

For NYC data, the additional cores don't seem to improve performance, but obviously objective testing will show the truth.

Also, this is all really slow on IE because it can't transfer workers (I think).  I need to talk to @shunter and expose a property on `FeatureDetection` so we can decide if we want to just do things synchronously in those cases. (not sure if that's the right answer though).